lib/dest/write-contents/index.js

```diff
@@ -12,6 +12,11 @@ var fo = require('../../file-operations');
 function writeContents(optResolver) {
 
   function writeFile(file, enc, callback) {
+    // Write it as a symlink
+    if (file.symlink) {
```

Comment:
Yes, I believe `file.isSymbolic()` returns true if and only if `file.stats && file.stats.isSymbolicLink()`. So that doesn't work for stat-less files.

Relying on `symlink` here should be fine, I think, but like I mentioned earlier I think that property should be attached in more places than is currently being done.

I had that figured out already though so I could quickly make a PR for that once this one is merged?